.github/workflows/python-package.yml

```diff
@@ -24,9 +24,10 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         if [ -f dev-requirements.txt ]; then pip install -r dev-requirements.txt; fi
+        pip install -e .
```

Comment:
Does this mean that the local pip-licenses package installation will be tested for success?

Reply:
I don't expect `pip install -e .` to fail. However this is necessary for relative imports inside the package.

Reply:
I got it. Since "Install dependencies" has already been completed, the following command may be fine.

```bash
pip install -e . --no-deps
```

Reply:
I don't think it's necessary. pip automatically checks if the dependencies are installed, thus `pip install -e .` will only link the `piplicenses` package.

See the [action log](https://github.com/raimon49/pip-licenses/pull/88/checks?check_run_id=1800977022) -> `Install dependencies` -> L124.

Reply:
My suggestion means that we can skip L123. But it is a trivial matter. We'll be fine as is.